### ISSUE TITLE
Don't nag about landing backed out patches

### ISSUE THREAD

**Issue by gcp** [mozilla/relman-auto-nag#810]
https://bugzilla.mozilla.org/show_bug.cgi?id=1546442

"There are some r+ patches which didn't land and no activity in this bug for 2 weeks."

The patches did actually land, but were backed out. There's a "Regressions" bug (which indeed also didn't have activity, but it doesn't seem the bot checked that).

**Comment by calixteman** [comment]
Probably the right way to do here, is to add "changes-planned" in Phabricator to your revision.
It isn't only for the bot but it'd help any people who read this bug to understand what the status of the patch.

**Comment by gcp** [comment]
1. We already have a "no-nag" to shut up the bot. 
2. If patches get backed out, obviously we'll try to fix them if possible. Readers can read the comments and status of the bug. If the info is only for the bot, see (1).
3. Regardless of the tag what the bot is saying is *still* outright wrong.

**Comment by marco-c** [comment]
> Readers can read the comments and see the status of the bug

This is true, but if you look at many bugs every day (e.g. if you are a release manager), being able to quickly glance the status of a bug without having to read comments is very beneficial.

> Regardless of the tag what the bot is saying is still outright wrong.

True, we should try to avoid this situation if it is common (if it isn't common maybe the added complexity is not worth it).

On a related note, perhaps we could automatically set changes-planned on Phabricator when a patch is backed out.

**Comment by gcp** [comment]
>On a related note, perhaps we could automatically set changes-planned on Phabricator when a patch is backed out.

Seems reasonable especially as currently the patch author would get a needinfo? on the backout.
